symengine/pow.cpp

```diff
@@ -87,6 +87,25 @@ int Pow::compare(const Basic &o) const
         return base_cmp;
 }
 
+bool exp_mul_helper(const RCP<const Basic> &b)
```

Comment:
How about this function returning -1, 0, 1,2,3 where -1 is when it is not what we want and the others `n` in the function beloe